### PR TITLE
Update user guide with images

### PR DESCRIPTION
**Pull request by melissaharijanto**
Since this is kind of a major user guide update, I think it would be good for more people to proofread, so feel free to comment if you have any suggestions / better wordings because I feel that a lot of these could be worded better :") 

Note: `&#8593;` is UTF-8 for &#8593;

### CODE REVIEW

**Review by donljh — APPROVED** [review]
Thanks for coming up with all the screenshots! Overall good work. We should read through the merged changes on the actual site after the merges to determine if we need to change anything

**Review by sohenze — APPROVED** [review]
Thanks for the screenshots!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -85,101 +91,134 @@ We have prepared a [Quick Start](#quick-start) section in this guide to equip yo
...
 4. Double-click the file to start the app. The GUI similar to the one below should appear in a few seconds. The application will already contain some sample
    data.<br>
    ![Ui](images/Ui.png)
 
 5. Head over to the [tutorial](#tutorial) section for a small walkthrough of some of TrackO's commands and use cases.
 
-6. If you'd like to skip the tutorial, head over to the [features](#features) section for some of the more technical 
-in-depth explanation of each command's format an use case.
+6. If you'd like to skip the tutorial, head over to the [features](#features) section for some of the more technical
+   in-depth explanation of each command's format a use case.
```
Comment (janelleljt) [inline docs/UserGuide.md]: I think here should be and?

**Review by janelleljt — APPROVED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (melissaharijanto) [reply docs/UserGuide.md]: added! thanks for pointing it out!

**Pull request merged**